### PR TITLE
commiting 2020.1 SDK changes

### PR DESCRIPTION
**Pull request by ShefaliBanthiaMyob**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,19 +1,20 @@
-Copyright 2018 MYOB Technology Pty Ltd
+The MIT License (MIT)
```
Comment (jaspermondie) [inline LICENSE]: Noticed the previous version was 2019, while this one is still 2013. Did we need to replace the LICENSE file?

File: appveyor.yml [hunk 2]
```diff
@@ -25,18 +25,23 @@ artifacts:
 deploy:
 - provider: NuGet
   api_key:
-    secure: Bt8SgvlZZ2V4hVkvaPnwBxiRy1uLw5WbksOUjMttEyramRkkooSEnF7GoPtVyZbI
+    secure: LlFtlduQCvv8/wLHFr7i9bLjEdmXklNHtl4X0dbU3qe6b2fduuSGzLlytZGuFr2a
```
[the remark this answers is not in the document]
Comment (ShefaliBanthiaMyob) [reply appveyor.yml]: yes. I have cross verified with perforce changes. API key got replaced with new one

File: appveyor.yml [hunk 3]
```diff
@@ -25,18 +25,23 @@ artifacts:
...
   skip_symbols: true
   artifact: Release
   on:
     branch: release
 - provider: NuGet
   api_key:
-    secure: Bt8SgvlZZ2V4hVkvaPnwBxiRy1uLw5WbksOUjMttEyramRkkooSEnF7GoPtVyZbI
+    secure: LlFtlduQCvv8/wLHFr7i9bLjEdmXklNHtl4X0dbU3qe6b2fduuSGzLlytZGuFr2a
```
[the remark this answers is not in the document]
Comment (ShefaliBanthiaMyob) [reply appveyor.yml]: yes. I have cross verified with perforce changes. API key got replaced with new one

File: LICENSE (hunk 1 above)
Comment (ShefaliBanthiaMyob) [reply LICENSE]: Modified file to 2020 version

**Pull request merged**